README.md

```diff
@@ -1 +1,22 @@
 # github_clone
+
+## By Aaron & Pearl
+
+You can view our live site: [https://agithubclone.netlify.app/](https://agithubclone.netlify.app/)
+
+## Installation 
+If you wish to run it locally, or run tests then do the following:
+1. Clone this repo (from main)
+2. Navigate the root directory of this app
+3. Install dependencies: **npm install**
+4. To build use: **npm run dev** (if you are on windows the link it opens wont work but if you navigate to localhost:8080 it will work)
+5. To run tests or coverage: **npm run test** or **npm run coverage**
+
+## What you can play around with:
+- Enter your username
+- Open up any repos you want to visit
+- Try all our links
+- Enter a username that does not exist
+- Edit the repo URL by putting in a random ID of your choice to see if it exists
+
+Thanks!
```

Comment:
Maybe add a section for wins/challenges and any bugs? (this is not us saying we found bugs!)